changelogs/fragments/993-vmware_vm_config_option.yml

```diff
@@ -0,0 +1,2 @@
+minor_changes:
+  - vmware_vm_config_option - add new module to get ESXi supported config option keys(hardware versions), supported guest IDs and VM recommended config options for specified guest ID (https://github.com/ansible-collections/community.vmware/issues/993).
```

Comment:
For the new module this is not required. Changelog entry will be created automatically for new module.

Reply:
Ok, I changed the description of this changelog, or do I just need to remove this changelog?

Reply:
I am OK with changed description.